### PR TITLE
docs(zh): translate Install docs into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
**What type of PR is this?**
translate Install docs into Chinese:
```
site/content/zh/latest/install/api.md
site/content/zh/latest/install/custom-cert.md
site/content/zh/latest/install/install-helm.md
site/content/zh/latest/install/install-yaml.md
```

### CODE REVIEW

File: site/content/zh/latest/install/install-helm.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+将 gRPC 的端口更改为 `18005`，将限流端口更改为 `18006`，并将日志记录级别更新为 `debug`。
+
+您可以通过以下命令使用 value.yaml 文件安装 Envoy Gateway。
+
+```shell
+helm install eg oci://docker.io/envoyproxy/gateway-helm --version v0.0.0-latest -n envoy-gateway-system --create-namespace -f values.yaml
+```
+
+{{% alert title="Helm Chart Values" color="primary" %}}
+如果您想了解 values.yaml 文件中的所有可用字段，请参阅 [Helm Chart Values](../api)。
```
Comment (shawnh2) [inline site/content/zh/latest/install/install-helm.md:117]:
```suggestion
如果您想了解 values.yaml 文件中的所有可用字段，请参阅 [Helm Chart Values](./api)。
```

this seems broke doc-lint in ci

Comment (wilsonwu) [reply site/content/zh/latest/install/install-helm.md]: Seems this is my bad for translation, I need to translate the api folder, I am working on it.

Comment (wilsonwu) [reply site/content/zh/latest/install/install-helm.md]: I am wrong, this is a mistake in EN doc, I updated EN link together.

File: site/content/zh/latest/install/install-helm.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
+```
+
+安装 GatewayClass、Gateway、HTTPRoute 和示例应用程序：
+
+```shell
+kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/latest/quickstart.yaml -n default
+```
+
+**注意**：[`quickstart.yaml`] 定义 Envoy Gateway 将侦听其全局可路由 IP 地址上端口 80 上的流量，
```
Comment (yuluo-yx) [inline site/content/zh/latest/install/install-helm.md:53]: `全局可路由 IP 地址上端口 80 上`  Is there a problem with the description?

File: site/content/zh/latest/install/install-yaml.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+此任务将引导您完成在 Kubernetes 集群中安装 Envoy Gateway。
+
+手动安装过程不允许像 [Helm 安装方法](./install-helm)那样对配置进行更多控制，
+因此如果您需要对 Envoy Gateway 安装进行更多控制，建议您使用 Helm。
+
+## 开始之前 {#before-you-begin}
+
+Envoy Gateway 设计为在 Kubernetes 中运行以进行生产。最重要的要求是：
+
+* Kubernetes 1.25 或更新
```
Comment (yuluo-yx) [inline site/content/zh/latest/install/install-yaml.md:15]: The `更新` is ambiguous, could it be changed to `Kubernetes 版本 >= 1.25`

Comment (wilsonwu) [reply site/content/zh/latest/install/install-yaml.md:15]: Yes, I will fix it.

File: site/content/zh/latest/install/install-helm.md (hunk 2 above)
Comment (wilsonwu) [reply site/content/zh/latest/install/install-helm.md:53]: The English is: `Envoy Gateway will listen for traffic on port 80 on its globally-routable IP address`

Do you think: `Envoy Gateway 将侦听 80 端口及其全局可路由 IP 地址的流量` is better?

Comment (yuluo-yx) [reply site/content/zh/latest/install/install-helm.md:53]: agree.

File: site/content/zh/latest/install/install-yaml.md (hunk 3 above)
Comment (zirain) [reply site/content/zh/latest/install/install-yaml.md:15]: `Kubernetest 1.25+ 版本` ?

**Review by shawnh2 — APPROVED** [review]
LGTM! Thanks

**Review by zhaohuabing — APPROVED** [review]
LGTM thanks!

**Pull request merged**